FirebaseStorage/Sources/FIRStorageUpdateMetadataTask.m

```diff
@@ -94,12 +92,13 @@ - (void)enqueue {
       }
       self->_fetcherCompletion = nil;
```

Comment:
Mostly curious, why the retain cycle is needed here? A comment on it would be useful to avoid such questions in the future.

Reply:
Maybe to avoid multiple calls? I'm not sure, but either way, it seems outside the scope of this PR.

Reply:
sure! You just removed the warning suppression so I thought you know why it's actually needed :)

Reply:
I removed the warning suppression because the warning suppression isn't needed :). Warnings don't get generated.